src/transformers/models/bert/modeling_bert.py

```diff
@@ -207,14 +208,10 @@ def forward(self, input_ids=None, token_type_ids=None, position_ids=None, inputs
 class BertSelfAttention(nn.Module):
     def __init__(self, config):
         super().__init__()
-        if config.hidden_size % config.num_attention_heads != 0 and not hasattr(config, "embedding_size"):
-            raise ValueError(
-                "The hidden size (%d) is not a multiple of the number of attention "
-                "heads (%d)" % (config.hidden_size, config.num_attention_heads)
-            )
 
         self.num_attention_heads = config.num_attention_heads
-        self.attention_head_size = int(config.hidden_size / config.num_attention_heads)
```

Comment:
These lines should not be touched please. Deleting those lines is not backwards compatible and  `Bert` is a highly used model. This could lead to a lot of problems